package.json

```diff
@@ -45,7 +52,13 @@
     "chai": "^4.2.0",
     "eslint": "^5.13.0",
     "eslint-config-google": "^0.12.0",
+    "eslint-config-prettier": "^4.0.0",
+    "eslint-plugin-prettier": "^3.0.1",
```

Comment:
again both of these can go if we just don't inherit the config (see first comment)